pagopa/io-app
# feat: [IOPID-1521,IOPID-1522,IOPID-1523] Integrate new DS in Profile section

## Short description
Integrate new DS in Profile section and Tax Code screen.

## List of changes proposed in this pull request
- Integration of new DS in Profile section;
- Implementation of new Banner in Profile section to guide the user to the new tax code section;
- In "Your Data" section:
   - Integration of new DS;
   - Addition of new Tax Code list item.
- Integration of new DS in Fiscal Code section

## Test
### E2E
>[!Tip]
>[![Run e2e tests](https://github.com/pagopa/io-app/actions/workflows/test-e2e.yml/badge.svg?branch=IOPID-1521-ds-profile-app-main-screen)](https://github.com/pagopa/io-app/actions/workflows/test-e2e.yml)

### Screens

<details><summary>iOS & Android</summary>
<p>

|iOS|Android|
|-|-|
|<video src="https://github.com/pagopa/io-app/assets/94987118/a382c880-a41c-49e9-b74a-9dc0165f67df" />|<video src="https://github.com/pagopa/io-app/assets/94987118/71d7d4e2-190f-4a39-8b5f-a6ebcac75b4f" />|

</p>
</details>

<details><summary>A11Y - iOS & Android</summary>
<p>

|iOS|Android|
|-|-|
|<video src="https://github.com/pagopa/io-app/assets/94987118/c7ad4b14-85b8-4678-acf6-eae57075b9ad" />|<video src="https://github.com/pagopa/io-app/assets/94987118/633d598a-7abc-4b3e-9b61-27ef89322bc3" />|

</p>
</details>

## How to test
- Navigate into the Profile section to check the new DS. The new Banner** should be visible on top of the screen.
- Clicking on the new Banner it will redirect you to the "Your Data" screen (even this one with the new DS applied).
- Once in the "Your Data" screen if you click the *cta* "Show" displayed in the Tax Code list item you will be redirected to the new Tax Code screen accordingly with the new DS directives.
>[!Important]
>Once in the Tax Code screen, if the brightness of your device is not at the maximum level, it will automatically be brought to the maximum level, in this case exiting this screen will restore the brightness of your device as previously set.

>[!Note]
>** Once the Banner is closed, it shouldn't appear anymore, even if you close and re-open the app. But don't worry, you will still be able to navigate into the "Your Data" screen by clicking on the dedicated item on the list.
